lib/core/enhanceError.js

```diff
@@ -16,12 +16,12 @@ module.exports = function enhanceError(error, config, code, request, response) {
     error.code = code;
   }
 
-  error.request = request;
+  error.request=request;
   error.response = response;
   error.isAxiosError = true;
 
-  error.toJSON = function() {
-    return {
+  error.toJSON = function( ) {
```

Comment:
format: style mismatch:
```suggestion
  error.toJSON =function () {
```

`16b835cf` ␣ at column 17 should be removed.
`8b4d12ee` ␣ should be inserted at column 26.
`16b835cf` ␣ at column 27 should be removed.